pkg/config/config.go

```diff
@@ -127,6 +132,7 @@ type HarvesterConfig struct {
 	HarvesterChartVersion  string            `json:"harvesterChartVersion,omitempty"`
 	MonitoringChartVersion string            `json:"monitoringChartVersion,omitempty"`
 	SystemSettings         map[string]string `json:"systemSettings,omitempty"`
+	ClusterNetworks        map[string]ClusterNetwork `json:"clusterNetworks,omitempty"`
```

Comment:
I guess there is a golint issue.

Reply:
golint only told me that need comment for that entry.